proposals/repo-to-tar/README.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Package Repository to Tar"
+authors: [ "Ashish Kumar <ashishndiitr@gmail.com>" ]
+status: "draft"
+approvers: [ ]
+---
+
+# <Package Repository to Tar>
```

Comment:
The main requirement for this proposal is to allow `kctrl` to release a Package Repository to an OCI Compliant tar file. Is this what we are describing here? From what I am reading I am not 100% sure.

Reply:
Changed this to be more in compatible with the change in imgpkg. Originally was writing a combined `kctrl + imgpkg` proposal, hence the old name.